application/config/applications/mapbender_user.yaml

```diff
@@ -1,128 +1,256 @@
 parameters:
     applications:
         mapbender_user:
-            title: Demo
+            title: Mapbender Demo
             screenshot: screenshot.png
-            description: Full desktop application showing most of the standard Mapbender elements
+            description: Mapbender demo application showing many elements in action
             published: true
+            persistentView: false
             template:  Mapbender\CoreBundle\Template\Fullscreen
+            custom_css: |
+                .popup {
+                border-color: white;
```

Comment:
I suggest a light grea border. Else when you open more than 1 popup it looks strange

                .popup {
                border: 1px solid #ccc;
                border-radius: 10px;
                }

Also line below head could be a bit lighter
                .popupHead {
                border-bottom: solid 1px #ccc;
                margin: 0 20px 20px 20px;
                padding: 20px 0 10px 0;
                }